packages/workflow/src/activities/generateOrAssignContentType.ts

```diff
@@ -38,25 +38,19 @@ export async function guessOrCreateDocumentType(payload: DSLActivityExecutionPay
     log.info("SelectDocumentType for object: " + objectId, { payload });
 
     const object = await client.objects.retrieve(objectId, "+text");
+
+    //Expects object.type to be null on first ingestion of content
+    //User can set type manually via "Change Content type" via change-type.ts
     if (object.type) {
         log.warn(`Object ${objectId} has already a type. Skipping type creation.`);
         return { status: "skipped", message: "Object already has a type: " + object.type.name };
     }
-
+    
     if (!object || (!object.text && !object.content?.type?.startsWith("image/") && !object.content?.type?.startsWith("application/pdf"))) {
         log.info(`Object ${objectId} not found or text is empty and not an image`, { object });
         return { status: "failed", error: "no-text" };
     }
 
-    if (object.type) {
```

Comment:
Just to confirm I understand your changes: this if-statement is removed because we can never reach this logic? That is, if the object type is null, then the activity would enter the previous if-statement at line 44, and skip the generation?

Reply:
Yes this is correct.

Reply:
awesome, looks good to me then 👍